data-miner/tables/Airbases.lua

```diff
@@ -1,4 +1,19 @@
 --- MISSION:default
+
+
+function generalPosObj(DCSpos)
+    lat, lon, alt = coord.LOtoLL(DCSpos)
+    return {
+        ['DCS'] = DCSpos,
+        ['World'] = {
+            ['lat'] = lat,
+            ['lon'] = lon,
+            ['alt'] = alt
+        }
+    }
+end
```

Comment:
maybe this can be a globally available generic function to we can convert all position data